components/BottomNavigation/tests/snapshot/MDCBottomNavigationLargeItemDialogViewSnapshotTests.m

```diff
@@ -0,0 +1,215 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <CoreGraphics/CoreGraphics.h>
+#import <XCTest/XCTest.h>
+
+#import "../../src/private/MDCBottomNavigationLargeItemDialogView.h"
+
+#import "MaterialSnapshot.h"
+
+static const CGFloat kImageHeight = 75;
+static const CGFloat kImageWidth = 75;
+
+@interface MDCBottomNavigationSystemDialogViewSnapshotTests : MDCSnapshotTestCase
+
+/** The dialog view undergoing testing. */
+@property(nonatomic, nonnull) MDCBottomNavigationLargeItemDialogView *dialogView;
+
+@end
+
+@implementation MDCBottomNavigationSystemDialogViewSnapshotTests
+
+- (void)setUp {
+  [super setUp];
+  self.dialogView = [[MDCBottomNavigationLargeItemDialogView alloc] initWithFrame:CGRectZero];
+
+  // Uncomment to rerecord golden images.
+  //  self.recordMode = YES;
+}
+
+- (void)tearDown {
+  _dialogView = nil;
```

Comment:
Nit: Please use `self.dialogView = nil;` since it's a property. Yay, properties!

Reply:
Done.